pkg/server/delta/v3/server.go

```diff
@@ -44,6 +54,210 @@ func NewServer(ctx context.Context, config cache.ConfigWatcher, callbacks Callba
 	}
 }
 
+func (s *server) processDelta(str stream.DeltaStream, reqCh <-chan *discovery.DeltaDiscoveryRequest, defaultTypeURL string) error {
+	streamID := atomic.AddInt64(&s.streamCount, 1)
+
+	// streamNonce holds a unique nonce for req-resp pairs per xDS stream. The server
+	// ignores stale nonces and nonce is only modified within send() function.
```

Comment:
We no longer ignore this, so let's add a TODO about stale nonce handling

Reply:
Cool, I'll update that

Reply:
Cool I'll update that comment